packages/contentstack-audit/package.json

```diff
@@ -76,7 +76,7 @@
     "prepack": "npm run build && oclif manifest && oclif readme",
     "test": "mocha --forbid-only \"test/**/*.test.ts\"",
     "version": "oclif readme && git add README.md",
-    "clean": "rm -rf ./lib ./node_modules tsconfig.tsbuildinfo oclif.manifest.json",
+    "clean": "rm -rf ./lib tsconfig.tsbuildinfo oclif.manifest.json",
```

Comment:
@cs-raj ,We are removing node_modules from each plugin. **./node_modules** does not need to be removed from the clean command

Reply:
added back